I can't compile this locally either.

```
heartsucker@pythagoras:~/code/heartsucker/rust-tuf (pr/170)$ rustc --version
rustc 1.33.0-nightly (09d6ab90e 2018-12-20)
heartsucker@pythagoras:~/code/heartsucker/rust-tuf (pr/170)$ cargo --version
cargo 1.33.0-nightly (2cf1f5dda 2018-12-11)
```

A small sample of the errors I'm getting.

```
error[E0658]: use of unstable library feature 'pin' (see issue #49150)
 --> /home/heartsucker/.cargo/registry/src/github.com-1ecc6299db9ec823/futures-core-preview-0.3.0-alpha.11/src/future/mod.rs:4:5
  |
4 | use core::pin::Pin;
  |     ^^^^^^^^^^^^^^
  |
  = help: add #![feature(pin)] to the crate attributes to enable

error[E0658]: use of unstable library feature 'pin' (see issue #49150)
 --> /home/heartsucker/.cargo/registry/src/github.com-1ecc6299db9ec823/futures-core-preview-0.3.0-alpha.11/src/future/future_obj.rs:4:27
  |
4 |     marker::{PhantomData, Unpin},
  |                           ^^^^^
  |
  = help: add #![feature(pin)] to the crate attributes to enable

error[E0658]: use of unstable library feature 'pin' (see issue #49150)
 --> /home/heartsucker/.cargo/registry/src/github.com-1ecc6299db9ec823/futures-core-preview-0.3.0-alpha.11/src/future/future_obj.rs:5:5
  |
5 |     pin::Pin,
  |     ^^^^^^^^
  |
  = help: add #![feature(pin)] to the crate attributes to enable

error[E0658]: use of unstable library feature 'pin' (see issue #49150)
  --> /home/heartsucker/.cargo/registry/src/github.com-1ecc6299db9ec823/futures-core-preview-0.3.0-alpha.11/src/future/future_obj.rs:22:13
   |
22 | impl<'a, T> Unpin for LocalFutureObj<'a, T> {}
   |             ^^^^^
   |
   = help: add #![feature(pin)] to the crate attributes to enable

error[E0658]: use of unstable library feature 'pin' (see issue #49150)
  --> /home/heartsucker/.cargo/registry/src/github.com-1ecc6299db9ec823/futures-core-preview-0.3.0-alpha.11/src/future/future_obj.rs:92:13
   |
92 | impl<'a, T> Unpin for FutureObj<'a, T> {}
   |             ^^^^^
   |
   = help: add #![feature(pin)] to the crate attributes to enable

error[E0658]: use of unstable library feature 'pin' (see issue #49150)
   --> /home/heartsucker/.cargo/registry/src/github.com-1ecc6299db9ec823/futures-core-preview-0.3.0-alpha.11/src/future/future_obj.rs:157:29
    |
157 |     F: Future<Output = T> + Unpin + 'a
    |                             ^^^^^
    |
    = help: add #![feature(pin)] to the crate attributes to enable

error[E0658]: use of unstable library feature 'pin' (see issue #49150)
   --> /home/heartsucker/.cargo/registry/src/github.com-1ecc6299db9ec823/futures-core-preview-0.3.0-alpha.11/src/future/future_obj.rs:171:50
    |
171 | unsafe impl<'a, T, F> UnsafeFutureObj<'a, T> for Pin<&'a mut F>
    |                                                  ^^^^^^^^^^^^^^
    |
    = help: add #![feature(pin)] to the crate attributes to enable

error[E0658]: use of unstable library feature 'pin' (see issue #49150)
   --> /home/heartsucker/.cargo/registry/src/github.com-1ecc6299db9ec823/futures-core-preview-0.3.0-alpha.11/src/future/future_obj.rs:211:54
    |
211 |     unsafe impl<'a, T, F> UnsafeFutureObj<'a, T> for Pin<Box<F>>
    |                                                      ^^^^^^^^^^^
    |
    = help: add #![feature(pin)] to the crate attributes to enable
```

How did you compile this?

--- a/.travis.yml
+++ b/.travis.yml
@@ -5,7 +5,7 @@ dist: trusty
 language: rust
 cache: cargo
 rust:
-  - nightly
+  - nightly-2019-01-09
 
 env:
   global:

--- a/appveyor.yml
+++ b/appveyor.yml
@@ -12,11 +12,11 @@ environment:
   matrix:
     # Rust - Nightly
     - TARGET: i686-pc-windows-gnu
-      RUST_VERSION: nightly
+      RUST_VERSION: nightly-2019-01-09
       BITS: 32
       MSYS2: 1
     - TARGET: x86_64-pc-windows-msvc
-      RUST_VERSION: nightly
+      RUST_VERSION: nightly-2019-01-09
       BITS: 64
 
 install:

--- a/src/client.rs
+++ b/src/client.rs
@@ -33,7 +33,7 @@
 //!     url::Url::parse("https://static.rust-lang.org/").unwrap(),
 //!     HttpClient::new(),
 //! )
-//! .user_agent_prefix("rustup/1.4.0")
+//! .user_agent("rustup/1.4.0")
 //! .build();
 //!
 //! let mut client = await!(Client::with_root_pinned(

--- a/src/repository.rs
+++ b/src/repository.rs
@@ -263,7 +263,7 @@ where
     url: Url,
     client: Client<C>,
     interchange: PhantomData<D>,
-    user_agent_prefix: Option<String>,
+    user_agent: Option<String>,
     metadata_prefix: Option<Vec<String>>,
     min_bytes_per_second: u32,
 }
@@ -279,7 +279,7 @@ where
             url: url,
             client: client,
             interchange: PhantomData,
-            user_agent_prefix: None,
+            user_agent: None,
             metadata_prefix: None,
             min_bytes_per_second: 4096,
         }
@@ -290,8 +290,8 @@ where
     /// Callers *should* include a custom User-Agent prefix to help maintainers of TUF repositories
     /// keep track of which client versions exist in the field.
     ///
-    pub fn user_agent_prefix<T: Into<String>>(mut self, user_agent_prefix: T) -> Self {
-        self.user_agent_prefix = Some(user_agent_prefix.into());
+    pub fn user_agent<T: Into<String>>(mut self, user_agent: T) -> Self {
+        self.user_agent = Some(user_agent.into());
         self
     }
 
@@ -313,9 +313,9 @@ where
 
     /// Build a `HttpRepository`.
     pub fn build(self) -> HttpRepository<C, D> {
-        let user_agent = match self.user_agent_prefix {
-            Some(ua) => format!("{} (rust-tuf/{})", ua, env!("CARGO_PKG_VERSION")),
-            None => format!("rust-tuf/{}", env!("CARGO_PKG_VERSION")),
+        let user_agent = match self.user_agent {
+            Some(user_agent) => user_agent,
+            None => "rust-tuf".into(),
         };
 
         HttpRepository {

